### PR TITLE
Fix doc and add details following to SLA PIM-7395

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/assets/upload-assets.md [hunk 1]
```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
...
 
-The PIM uses the filename to know for which asset the file have to be used.
+The PIM uses the filename to know for which asset the file have to be used and if this asset is localized or not.
 
-**Example 1**  
+**Example 1 with a not localized asset**  
 If the filename is `main_picture_S1263547.jpg`, then the PIM will check:
-- If the asset with the `main_picture_S1263547` code already exists, the PIM will update the asset with the `main_picture_S1263547` code by importing the `main_picture_S1263547.jpg` file as its reference file (and generate the variation files accordingly)
-- If the asset doesn’t already exist, the PIM will create a new asset with the `main_picture_S1263547` code and with the file as the reference file, so that the variation files can be generated
+- If the asset with the `main_picture_S1263547` code already exists, the PIM will update the asset with the `main_picture_S1263547` code by importing the `main_picture_S1263547.jpg` file as its reference file (and generate the variation files accordingly).
```
Comment (LaureBro) [inline content/md/assets/upload-assets.md]: as its new reference file (and will regenerate the variation files...)

File: content/md/assets/upload-assets.md [hunk 2]
```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
...
+- If the asset doesn’t already exist, the PIM will create a new asset with the `main_picture_S1263547` code and with the file as the reference file, so that the variation files can be generated.
 
-**Example 2**  
-Another example, if the filename is `asset_en_US.pdf` and the asset with the `asset` code already exists, then the PIM will check:
-- If the asset is localized, the PIM will import the file as the reference for the given locale code at the end of the filename (`en_US`)
-- If the asset is not localized, the PIM will display an error message in red, to inform that the PIM does not know what to do with this file
+**Example 2 with a localized asset**  
+Another example, if the filename is `akene-en_US.pdf`, then the PIM will check:
+- If the asset with the `akene` code already exists and is localized, the PIM will update the asset with the `akene` code by importing the `akene-en_US.pdf` file as the reference file for the given locale code at the end of the filename (`en_US`).
```
Comment (LaureBro) [inline content/md/assets/upload-assets.md]: i'll write again "and if it is localized"

File: content/md/assets/upload-assets.md [hunk 3]
```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
...
+- If the asset with the `akene` code already exists and is not localized, the PIM will display an error message in red.
```
Comment (LaureBro) [inline content/md/assets/upload-assets.md]: same as above: 
If the asset with the `akene` code already exists and **if it** is not localized

**Comment by LaureBro** [comment]
some comments :)

File: content/md/assets/upload-assets.md [hunk 4]
```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
 
 Mass upload assets is pretty convenient especially if you want to update your asset catalog with external sources: photo shooting, new collection, etc...
 
-The PIM uses the filename to know for which asset the file have to be used.
+The PIM uses the filename to know for which asset the file have to be used and if this asset is localized or not.
 
-**Example 1**  
+**Example 1 with a not localized asset**
```
[the remark this answers is not in the document]
Comment (Delphineray) [reply content/md/assets/upload-assets.md]: ok

File: content/md/assets/upload-assets.md (hunk 1 above)
Comment (Delphineray) [reply content/md/assets/upload-assets.md]: done

File: content/md/assets/upload-assets.md (hunk 2 above)
Comment (Delphineray) [reply content/md/assets/upload-assets.md]: done

File: content/md/assets/upload-assets.md (hunk 3 above)
Comment (Delphineray) [reply content/md/assets/upload-assets.md]: done

File: content/md/assets/upload-assets.md [hunk 5]
```diff
@@ -11,17 +11,22 @@ related: work-with-assets, assets-transformation
 
 Mass upload assets is pretty convenient especially if you want to update your asset catalog with external sources: photo shooting, new collection, etc...
```
Comment (LaureBro) [inline content/md/assets/upload-assets.md]: Should be Asset mass upload

**Pull request merged**